whylog/config/investigation_plan.py

```diff
@@ -50,9 +50,26 @@ def compare_with_bound(self, bound, super_parser_groups):
         Basing on super_parser_groups extracted from line, returns information
         how relative to bound this line is.
         """
-        #TODO: remove mock
+        # This implementation assume that super_parser_groups length equals 1 or 0
+        # TODO implementation for longer super_parser_groups_list
+        if not super_parser_groups:
+            return self._compare_with_undefined_bound(bound)
+        group_type, group_value = super_parser_groups[0]
+        type_bounds = self._search_ranges.get(group_type)
+        if type_bounds is None:
```

Comment:
what happens here? I suspect we silently make super_parser not really do anything if it is mis-configured.